test/system/libpod_suite_test.go

```diff
@@ -0,0 +1,217 @@
+package system
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	. "github.com/containers/libpod/test/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	PODMAN_BINARY string
+	GLOBALOPTIONS = []string{"--cgroup-manager",
+		"--cni-config-dir",
+		"--config", "-c",
+		"--conmon",
+		"--cpu-profile",
+		"--log-level",
+		"--root",
+		"--tmpdir",
+		"--runroot",
+		"--runtime",
+		"--storage-driver",
+		"--storage-opt",
+		"--syslog",
+	}
+	PODMAN_SUBCMD = []string{"attach",
+		"commit",
+		"container",
+		"build",
+		"create",
+		"diff",
+		"exec",
+		"export",
+		"history",
+		"image",
+		"images",
+		"import",
+		"info",
+		"inspect",
+		"kill",
+		"load",
+		"login",
+		"logout",
+		"logs",
+		"mount",
+		"pause",
+		"ps",
+		"pod",
+		"port",
+		"pull",
+		"push",
+		"restart",
+		"rm",
+		"rmi",
+		"run",
+		"save",
+		"search",
+		"start",
+		"stats",
+		"stop",
+		"tag",
+		"top",
+		"umount",
+		"unpause",
+		"version",
+		"wait",
+		"h",
+	}
```

Comment:
Aren't a lot of these slices defined elsewhere in Podman?  Can we define them in one place rather than duplicating here?

Reply:
Just checked the code under cmd/podman/, seems all commands are unexported in the package. So we may not be able to read them from there. And as system test are more like a user side test, we may want to ignore the influence from low-level code from podman itself.
But I feel I may misunderstand what you are suggesting. Can you make it more clear? Like what file or scripts do you reference here? Thanks a lot.